src/electron/routing_service.ts

```diff
@@ -138,7 +168,11 @@ export class WindowsRoutingService implements RoutingService {
                       new errors.UnsupportedRoutingTable(msg) :
                       new errors.ConfigureSystemProxyFailure(msg));
             }
-            resolve(response);
+            if ('returnValue' in response) {
```

Comment:
In what scenario does this happen?

Reply:
This is for returning the name of tun device from the daemon.

Reply:
Would be good to have a comment explaining the difference between `response.returnValue` and `data`.